src/mock.jl

```diff
@@ -34,16 +34,16 @@ function get_alternate(pe::PatchEnv, target, args...)
         m, f = dispatch(pe.mapping[target], args...)
 
         if pe.debug
-            @info "calling mocked method: $m"
+            if m !== nothing
+                @info "calling mocked method: $m $f($(args...))"
+            else
+                target_m, _ = dispatch([target], args...)
+                @info "calling original method: $target_m $target($(args...))"
```

Comment:
Seem like too much info to include but 🤷‍♂️.

(Bother here and above)


Also this will catch fire pre1.3 if any of those interpolated things are `nothing`.

Maybe use the kwarg form?

 `@info "calling mocked method" m f args`

Etc

Reply:
I was working on this and the changes started so snowball. I'll get back to this next week

Reply:
I changed how these messages were being displayed. Instead of showing the `repr` string of a value I'm now showing the type being passed in. I think this is better as it is possible that the string representation will not reveal the type being used.